src/Sylius/Bundle/ShippingBundle/spec/Form/Type/ShippingMethodChoiceTypeSpec.php

```diff
@@ -13,24 +13,20 @@
 
 use PhpSpec\ObjectBehavior;
 use Prophecy\Argument;
-use Sylius\Component\Registry\PrioritizedServiceRegistryInterface;
 use Sylius\Component\Registry\ServiceRegistryInterface;
 use Sylius\Component\Resource\Repository\RepositoryInterface;
-use Sylius\Component\Shipping\Model\ShippingSubjectInterface;
-use Sylius\Component\Shipping\Resolver\CompositeMethodsResolverInterface;
 use Sylius\Component\Shipping\Resolver\MethodsResolverInterface;
 use Symfony\Bridge\Doctrine\Form\DataTransformer\CollectionToArrayTransformer;
 use Symfony\Component\Form\AbstractType;
 use Symfony\Component\Form\FormBuilderInterface;
-use Symfony\Component\OptionsResolver\OptionsResolver;
 
 /**
  * @author Arnaud Langlade <arn0d.dev@gamil.com>
  */
 class ShippingMethodChoiceTypeSpec extends ObjectBehavior
 {
     function let(
-        CompositeMethodsResolverInterface $compositeMethodsResolver,
+        MethodsResolverInterface $compositeMethodsResolver,
```

Comment:
This variable is called `$compositeMethodsResolver` but the interface is now `MethodsResolverInterface`. It's just a small thing but this now makes it seem like a composite methods resolver is expected to be injected in, even though that won't necessarily be true.

Similar occurrences elsewhere.